Infrastructure/Persistance/Relational/Configurations/DeckCardConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+
+namespace Infrastructure.Persistance.Relational.Configurations
+{
+    public class DeckCardConfiguration : IEntityTypeConfiguration<DeckCard>
```

Comment:
Lille ting, men det kunne være fedt at eksplicit få sat hvad default værdien for isPublic er her, og og med at den ikke bliver sat i repository

Reply:
Tænker du på Deck config, eller mener du faktisk at den skal sættes på DeckCard? xd